api/net/Areas/Subscriber/Controllers/ContentController.cs

```diff
@@ -324,7 +324,7 @@ public async Task<IActionResult> ShareAsync(long contentId, int colleagueId, int
             NotificationId = notification.Id,
             ContentId = contentId,
             RequestorId = user.Id,
-            To = colleague.Email,
+            To = colleague.PreferredEmail != "" ? colleague.PreferredEmail : colleague.Email,
```

Comment:
You don't need to change, but this is one of those areas where I prefer `!String.IsNullOrWhiteSpace(colleague.PreferredEmail)`

Reply:
Thank you for the heads up ... it really looks better.